tiledb/sm/serialization/tiledb-rest.capnp

```diff
@@ -1196,6 +1196,15 @@ struct BufferedChunk {
   # the size in bytes of the intermediate chunk
 }
 
+struct ArrayFragmentsList {
```

Comment:
Honestly I don't really understand why these structures are not part of ArrayDirectory, but I don't know the requirements of this feature, I'm guessing we need them to work outside refactored array open, correct?